### ISSUE TITLE
Example 1: radiative flux across an atmospheric layer

### ISSUE THREAD

**Issue by gwemon** [i-adopt/complex_property_analysis#1]
Exercise: tp decompose this observable property into the atomic components presented in https://docs.google.com/spreadsheets/d/1MxoUGO27ul1aa8h1xD5hRBZXAwGXsL7NSB8ZUMX1cRc/edit#gid=559215109:

provisional "code" | keywords | definition
-- | -- | --
A | quantity/quantity kind/quality/trait (WHAT theme) | Describes a measurable or qualifiable characteristic of "C"; if measured it determines the dimension of what is being observed
B | stats (WHAT theme) | Modification of "A" that originates from post-processing of the observation
C | target object (WHAT theme) | Any biological organism, chemical, or physical or chemical phenomenon that is being observed
D | matrices/substrate (WHAT and WHERE theme) | The environment in which the measurement is taken
E | method-related (HOW theme) | Procedure used to obtain a value for the combination of A and C
F | Constraint or "Other" | Condition in which "C" is at the time of the observation or any other elements that do not fit into the categories above

**Comment by gwemon** [comment]
Here is my attempt (to be reviewed by group):
A = radiative flux density
(see https://glossary.ametsoc.org/wiki/Radiative_flux_density and https://en.wikipedia.org/wiki/Radiative_flux#cite_note-2)
C= electromagnetic radiation
D= atmosphere
If this is the radiative flux across a specific atmospheric layer then the matrix term will need to be reflect this.

**Comment by graybeal** [comment]
Do you have a definition of this entity?  It is impossible to know how it should be broken down without knowing, in particular, the meaning of 'across an atmospheric layer'.

**Comment by gwemon** [comment]
All good points @graybeal 
This example was submitted to us during the RDA P16 I-ADOPT session and we do not have any other info. However the fact we have these questions and this discussion, including saying that we cannot define the term because some key information is missing or is ambiguous, is very useful too. Maybe people familiar with this type of measurements will be able to clarify what is needed here.
I think you are right that the defintion of "matrix" as it stands is confusing. "D" (the "matrix") is directly related to either the property observed or the target object. It is independent from the location of the measuring device. So the matrix of the property of an object measured by a remote sensing device (i.e. part of the method) is whatever environment that property of that object relates to (or characterises) at the time of the measurement. So, in your example, if the property measured relates to an object in the sea bed then the sea bed should be the matrix. If the object is the sea bed itself then we are in the situation where the "matrix" is the "target object". It would be good to gather real examples that help us illustrate (and test)  this modellisation in a number of situations.
So would the following definition for D work "The environment to which the observed property relates or in which the target object is embedded"?

**Comment by mabablue** [comment]
well, I see a problem with the overlap between A and C, both containing the term radiation...

**Comment by gwemon** [comment]
> well, I see a problem with the overlap between A and C, both containing the term radiation...

Yes I was wondering about this... Is specifying electromagnetic radiation needed here? One advantage of making it explicit I think, is that we then have a way of selecting "electromagnetic radiation" as the target object and get all the quantities related to it...

**Comment by mariutzica** [comment]
@mabablue the term power flux density could be used here instead; I’ve found that sometimes it is not possible to untangle all of the objects of interest from the property terminology without losing the domain meaning, so it may be necessary to be redundant  (e.g. hydraulic conductivity, electron affinity)

**Comment by mariutzica** [comment]
@gwemon I agree; there is definitely an advantage to explicitly stating that radiative flux is about radiation

**Comment by mabablue** [comment]
check the google sheet version here.. https://docs.google.com/spreadsheets/d/1lnyDKdMuJdokmr2II7mPpZ60yxjo9S42qHyy1YxAczc/edit#gid=2025491598

**Comment by japamment** [comment]
I agree with @graybeal that this quantity is not well defined. In CF standard names we would write it as something like radiative_flux_in_air (units would be W m-2) and defined as 'Radiative flux is the sum of shortwave and longwave radiative fluxes. In accordance with common usage in geophysical disciplines, "flux" implies per unit area, called "flux density" in physics.' However, this is an incomplete definition - CF would require a direction (upwelling/downwelling). Alsooes the quantity assume that there is no abosorption or reflection of radiation within the layer so the radiation is passing straight through? As it stands we would not create this standard name!
